framework/arcane-framework/build.sbt

```diff
@@ -5,5 +5,20 @@ ThisBuild / scalaVersion := "3.6.1"
 lazy val root = (project in file("."))
   .settings(
     name := "arcane-framework",
-    idePackagePrefix := Some("com.sneaksanddata.arcane.framework")
+    idePackagePrefix := Some("com.sneaksanddata.arcane.framework"),
```

Comment:
you are missing `scalaVersion` - should be 3.5
(let's not use Scala 2 - v3 has been out for a while, so I suggest we go with it right away)
https://www.scala-sbt.org/1.x/docs/Howto-Scala.html
https://www.scala-lang.org/download/all.html

Reply:
done